airflow/providers/google/cloud/hooks/cloud_sql.py

```diff
@@ -520,12 +520,12 @@ def _get_credential_parameters(self, session: Session) -> List[str]:
         connection = session.query(Connection). \
             filter(Connection.conn_id == self.gcp_conn_id).first()
         session.expunge_all()
-        if GCP_CREDENTIALS_KEY_PATH in connection.extra_dejson:
+        if connection.extra_dejson.get(GCP_CREDENTIALS_KEY_PATH):
             credential_params = [
                 '-credential_file',
                 connection.extra_dejson[GCP_CREDENTIALS_KEY_PATH]
             ]
-        elif GCP_CREDENTIALS_KEYFILE_DICT in connection.extra_dejson:
+        elif connection.extra_dejson.get(GCP_CREDENTIALS_KEYFILE_DICT):
```

Comment:
Is this change necessary? Personally I think the current version is more accurate

Reply:
I was trying to resolve this issue https://issues.apache.org/jira/browse/AIRFLOW-5801.

Reply:
@tomek these elements exist but are empty when you create JSON through the form in the web UI.

Reply:
Oh, got it!